app/helpers/application_helper.rb

```diff
@@ -4,41 +4,46 @@ def asset_manager_initial_state
     portfolio = current_asset_manager.portfolio
 
     {
-      buildings: ActiveModel::Serializer::CollectionSerializer.new(
-        portfolio.buildings, each_serializer: BuildingSerializer,
-        scope: {user_id: current_asset_manager.id,
-                user_type: 'AssetManager'}
-      ),
-      userType: 'AssetManager'
+        user: current_asset_manager,
+        buildings: ActiveModel::Serializer::CollectionSerializer.new(
+            portfolio.buildings, each_serializer: BuildingSerializer,
+            scope: {user_id: current_asset_manager.id,
+                    user_type: 'AssetManager'}
+        ),
+        portfolios: portfolio,
+        categories: current_asset_manager.categories,
+        userType: 'AssetManager'
     }
   end
 
   def building_op_initial_state
     # Initial state here
     {
-      buildings: ActiveModel::Serializer::CollectionSerializer.new(
-       current_building_operator.buildings, each_serializer: BuildingSerializer,
-       scope: {user_id: current_building_operator.id,
-               user_type: 'BuildingOperator'}
-      ),
-      userType: 'BuildingOperator'
+        user: current_building_operator,
+        buildings: ActiveModel::Serializer::CollectionSerializer.new(
+            current_building_operator.buildings, each_serializer: BuildingSerializer,
+            scope: {user_id: current_building_operator.id,
+                    user_type: 'BuildingOperator'}
+        ),
+        categories: current_building_operator.categories,
+        userType: 'BuildingOperator'
     }
   end
 
   def rmi_user_initial_state
     {
-      portfolios: Portfolio.all,
-      buildings: ActiveModel::Serializer::CollectionSerializer.new(
-        Building.all, each_serializer: BuildingSerializer,
-        scope: {user_id: current_rmi_user.id,
-                user_type: 'RmiUser'}
-      ),
-      building_types: ActiveModel::Serializer::CollectionSerializer.new(
-        BuildingType.all, each_serializer: BuildingTypeSerializer,
-        scope: {user_id: current_rmi_user.id,
-                user_type: 'RmiUser'}
-      ),
-      userType: 'RMIUser'
+        portfolios: Portfolio.all,
```

Comment:
why no `user: current_rmi_user` in this hash?

Reply:
just havent put it in there yet, i thought we would load that in when someone needed it